src/Middleware/SpaServices.Extensions/src/AngularCli/AngularCliMiddlewareOptions.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.SpaServices.AngularCli
+{
+    /// <summary>
+    /// Describes options for starting the Angular CLI process
+    /// </summary>
+    public class AngularCliMiddlewareOptions
+    {
+        /// <summary>
+        /// The name of the script in your package.json file that launches the Angular CLI process.
+        /// </summary>
+        public string npmScript;
```

Comment:
Shouldn't this be 
```
public string NpmScript;
```
?
(I don't know why GitHub won't let me make that a suggestion).